Makefile

```diff
@@ -38,21 +40,19 @@ docker-down:
 	docker compose down -v --remove-orphans && docker compose rm
 
 install-katana:
-	cargo install --git https://github.com/dojoengine/dojo --locked --rev be16762 katana
+	cargo install --git https://github.com/dojoengine/dojo --locked --rev dfe390 katana
 
-run-katana: install-katana
+katana-genesis:
 	rm -fr .katana/ && mkdir .katana
-	katana --disable-fee --chain-id=KKRT --dump-state .katana/dump.bin & echo $$! > .katana/pid
+	cargo run --bin genesis --features testing
 
-kill-katana:
-	kill -2 `cat .katana/pid` && rm -fr .katana/pid
+run-katana: install-katana katana-genesis
```

Comment:
since we run the katana-genesis binary before running katana, we don't need deploy-kakarot anymore in most of the scenarios, let's just check it's clear for users in the docs
alternatively, a short comment above run-katana should help!

`starts a local katana sequencer with kakarot in its genesis`

Reply:
I would still keep the deploy-kakarot no? run-katana is specific to tests.

Reply:
yes yes let's keep it, just add indications so that people dont run make run-katana && make deploy-kakarot